tests/test_filtering.py

```diff
@@ -62,6 +64,13 @@ class WidgetListView(GenericModelView):
 
         filtering = Filtering(
             color=operator.eq,
```

Comment:
I feel like this tuple based api is good only for simple cases, when the filter is complicated, passing first an argument and then a dictionary becomes slightly awkward, it might be more readable to just use `ColumnFilterField` directly? I think that's a less surprising api?